holmes/plugins/toolsets/http/http_toolset.py

```
@@ -0,0 +1,425 @@
+            description=(
+                "Make HTTP requests to whitelisted API endpoints. "
+                "Use this tool for APIs that don't have a dedicated toolset. "
+                "IMPORTANT: Always prefer more specific toolsets when available "
+                "(e.g., use grafana tools for Grafana, prometheus tools for Prometheus). "
+                "This tool is for general-purpose HTTP access to configured endpoints."
+            ),
+            parameters=parameters,
+        )
+        self._toolset = toolset
+
+    def _invoke(self, params: dict, context: ToolInvokeContext) -> StructuredToolResult:
```

Ruff (0.14.13)

Unused method argument: `context`

(ARG002)